src/tbots.py

```diff
@@ -159,7 +171,7 @@
     # because it relies on --debug_simulator, --debug_blue_full_system and
     # --debug_yellow_full_system prompts the user to run the command under gdb
     # instead. So we only run_under gdb if its _not_ a thunderscope debug command
-    if args.action in "run" and args.debug_build:
+    if args.action in "run" and args.debug_build and args.run_under_gdb:
         if (
```

Comment:
I'm honestly not sure if `run_under_gdb` is useful. I tried to use it with unix_full_system but we have no way of passing in the runtime dir through `tbots.py` so I'm not sure where I'd be able to use it.

Reply:
okay I've removed it. Recently I found that the easiest way to debug is to launch build Thunderscope with debug symbols (now `./tbots.py run thunderscope -d`) and attach a remote debugger to either of the fullsystems (either through CLion or `gdb -p <pid>`). This seems more straight forward to me than running one of the fullsystem binaries separately.